package.json

```diff
@@ -11,7 +11,7 @@
     "start": "NODE_OPTIONS=--max_old_space_size=8192 ./build.js --no-clear -run -watch --settings settings.json",
     "start-dev-db": "./build.js --no-clear -run --watch --mongoUrlFile ../LessWrong-Credentials/connectionStrings/dev-db.txt --settings ../LessWrong-Credentials/settings-local-dev-devdb.json",
     "start-prod-db": "./build.js --no-clear -run --watch --mongoUrlFile ../LessWrong-Credentials/connectionStrings/prod-db.txt --settings ../LessWrong-Credentials/settings-local-dev-proddb.json",
-    "start-local-db": "./build.js --no-clear -run --watch --mongoUrl mongodb://127.0.0.1:27017/?directConnection=true --settings ./settings-test.json",
+    "start-local-db": "PORT=4000 ./build.js --no-clear -run --watch --mongoUrl mongodb://127.0.0.1:27017/?directConnection=true --settings ./settings-test.json",
```

Comment:
Is this meant to be here? You could probably just add a separate command, like we have `ea-start` and `ea-start-xpost-dev`.

Reply:
Yeah I haven't decided how to structure this yet - we don't use any flows involving the local db much so modifying this command is fine for now, but long-term the whole thing could stand to be cleaned up a bit.